admin_scripts/clear_redis_cache.py

```
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
+"""
+Redis Cache Management Tool for Crash Monitor.
+
+This script provides utilities to manage the Redis cache for the crashed-backend application.
+"""
+
+import argparse
+import sys
+import os
+import time
+from urllib.parse import urlparse
```

Ruff (0.11.9)

`urllib.parse.urlparse` imported but unused

Remove unused import: `urllib.parse.urlparse`

(F401)